memdown.js

```diff
@@ -136,6 +136,7 @@ MemDOWN.prototype._open = function (options, callback) {
 }
 
 MemDOWN.prototype._put = function (key, value, options, callback) {
+  if (typeof value === 'undefined' || value === null) value = ''
```

Comment:
Certainly could be generalised to `if (value == null)` (if you didn't want other falsy values)